group/assignment5/UnknownAlphabet_test.java

```diff
@@ -0,0 +1,94 @@
+import static org.junit.Assert.*;
+
+import java.util.LinkedList;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Test;
+
+
+public class UnknownAlphabet_test {
```

Comment:
In Java we usually use camelCase, I would rename this UnknownAlphabetTest

Reply:
Thanks, I'll change it now.